ts/sagas/profile.ts

```diff
@@ -64,67 +64,73 @@ function* createOrUpdateProfileSaga(
   >["createOrUpdateProfile"],
   action: ActionType<typeof profileUpsert["request"]>
 ): Iterator<Effect> {
-  // Get the current Profile from the state
-  const profileState: ReturnType<typeof profileSelector> = yield select<
-    GlobalState
-  >(profileSelector);
+  try {
```

Comment:
can we reduce the scode of this try/catch block? only network requests will throw exception (e.g. `createOrUpdateProfile`)